dispatch/dispatch.go

```diff
@@ -207,8 +204,13 @@ func (d *Dispatcher) Groups(routeFilter func(*Route) bool, alertFilter func(*typ
 			groups = append(groups, alertGroup)
 		}
 	}
-
 	sort.Sort(groups)
+	for i := range groups {
+		sort.Sort(groups[i].Alerts)
```

Comment:
great idea

Reply:
It was very much needed for the tests to pass :)